xclim/indicators/atmos/_conversion.py

```diff
@@ -38,6 +39,15 @@ class Converter(Indicator):
     compute=indices.humidex,
 )
 
+heat_index = Converter(
+    identifier="heat_index",
+    units="C",
+    standard_name="air_temperature",
+    long_name="heat index",
+    description="Days in which the human body perceives temperature is hot, which combines relative humidity with the air temperature.",
```

Comment:
I'm not sure I understand this description (and the abstract). It says "Days [...]", but the output is a temperature-like value, no?

Reply:
I've corrected this now.